partition_snapshot_row_cursor.hh

```diff
@@ -473,10 +473,10 @@ public:
 
     // Can be called only when cursor is valid and pointing at a row.
     clustering_row row() const {
-        clustering_row cr(key());
-        consume_row([&] (const deletable_row& row) {
-            cr.apply(_schema, row);
-        });
+        clustering_row cr(key(), deletable_row(_schema, _current_row[0].it->row()));
```

Comment:
This breaks if _current_row.size() == 0. Is it impossible?

Reply:
I didn't look too deep, but if the comments are consistent, then it's impossible.

Exhibit 1 (cursor is pointing at a row iff !_current_row.empty()):
```
    // Call only when valid.
    // Returns true iff the cursor is pointing at a row.
    bool at_a_row() const { return !_current_row.empty(); }
```
Exhibit 2 (row() is called only if cursor is pointing at a row):
```
    // Can be called only when cursor is valid and pointing at a row.
    clustering_row row() const {
```

Reply:
Ok. Please add this as comments in case a future reader asks the same question.